docs/resources/Channel.md

```diff
@@ -562,15 +562,17 @@ An object that represents a tag that is able to be applied to a thread in a `GUI
 
 ###### Forum Tag Structure
 
-| Field      | Type       | Description                                                                                                    |
-| ---------- | ---------- | -------------------------------------------------------------------------------------------------------------- |
-| id         | snowflake  | the id of the tag                                                                                              |
-| name       | string     | the name of the tag (0-20 characters)                                                                          |
-| moderated  | boolean    | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
-| emoji_id   | ?snowflake | the id of a guild's custom emoji \*                                                                            |
-| emoji_name | ?string    | the unicode character of the emoji \*                                                                          |
-
-\* At most one of `emoji_id` and `emoji_name` may be set.
+| Field      | Type      | Description                                                                                                    |
+| ---------- | --------- | -------------------------------------------------------------------------------------------------------------- |
+| id         | snowflake | the id of the tag                                                                                              |
+| name \*    | string    | the name of the tag (0-20 characters)                                                                          |
+| moderated  | boolean   | whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
+| emoji_id   | snowflake | the id of a guild's custom emoji \*\*                                                                          |
+| emoji_name | ?string   | the unicode character of the emoji \*\*                                                                        |
```

Comment:
```suggestion
| id         | snowflake | ID of the tag                                                                                                  |
| name \*    | string    | Name of the tag (0-20 characters)                                                                              |
| moderated  | boolean   | Whether this tag can only be added to or removed from threads by a member with the `MANAGE_THREADS` permission |
| emoji_id   | snowflake | ID of a guild's custom emoji \*\*                                                                              |
| emoji_name | ?string   | Unicode character of the emoji \*\*                                                                            |
```

Reply:
while im looking at this, just updating the formatting for this table even though it wasn't part of your PR

Reply:
most tables use lowercase descriptions so this would be inconsistent

Reply:
@advaith1 they won't for long 😏 but fair, i will take care of in a big PR to refactor tables